src/current/v25.3/citext.md

```diff
@@ -0,0 +1,126 @@
+---
+title: CITEXT
+summary: The CITEXT data type stores case-insensitive text values.
+toc: true
+docs_area: reference.sql
+---
+
+The `CITEXT` [data type]({% link {{ page.version.version }}/data-types.md %}) stores case-insensitive strings.
+
+All `CITEXT` values are folded to lowercase before comparison. This is handled internally with the [`lower()`]({% link {{ page.version.version }}/functions-and-operators.md %}#string-and-byte-functions) function.
+
+For example, the following comparison evaluates to `true`:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT 'Roach'::CITEXT = 'roach'::CITEXT;
+~~~
+
+~~~
+  ?column?
+------------
+     t
+~~~
+
+With `CITEXT`, equality operators (`=`, `!=`, `<>`), ordering operators (`<`, `>`, etc.), and [`STRING` functions]({% link {{ page.version.version }}/functions-and-operators.md %}#string-and-byte-functions), treat values as case-insensitive by default. Refer to the [example](#example).
+
+{{site.data.alerts.callout_info}}
+Unlike in PostgreSQL, `CITEXT` on CockroachDB is case-insensitive for all Unicode characters that have a defined uppercase/lowercase mapping (e.g., `'É' = 'é'`).
+{{site.data.alerts.end}}
+
+Aside from comparisons, `CITEXT` behaves like [`STRING`]({% link {{ page.version.version }}/string.md %}).
+
+## Syntax
+
+To declare a `CITEXT` column, use the type name directly in your `CREATE TABLE` statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE logins (
+    name CITEXT PRIMARY KEY,
+    email TEXT NOT NULL
+);
+~~~
+
+## Size
+
+As with `STRING`, `CITEXT` values should be kept below 64 KB for best performance. Because `CITEXT` values are folded to lowercase on every comparison, `CITEXT` columns and indexes consume marginally more CPU and memory than their `STRING` equivalents, especially on write-heavy workloads.
+
+## Collations
```

Comment:
IMO the paragraph is more of a "NOTE" than something deserving it's own "Collations" section.